server/routes/serviceProviderReferrals/serviceProviderReferralsController.ts

```diff
@@ -84,7 +85,7 @@ export default class ServiceProviderReferralsController {
   }
 
   async showMyCasesDashboard(req: Request, res: Response): Promise<void> {
-    if (!config.features.spDashboardPaginationFlow) {
+    if (!FeatureFlagService.enableForUser(res.locals.user, config.dashboards.serviceProvider.filteredUserPercentage)) {
```

Comment:
i wonder if this would be easier to read if the order of the branches were swapped and we can remove the `!`?